pkg/util/util.go

```diff
@@ -135,3 +136,43 @@ func IsSPDKTgtReady(timeout time.Duration) bool {
 	}
 	return false
 }
+
+// FindProcessByCmdline finds the process with matching cmdline
+func FindProcessByCmdline(cmdline string) (*os.Process, error) {
+	procDir, err := os.Open("/proc")
+	if err != nil {
+		return nil, err
+	}
+	defer procDir.Close()
+
+	processes, err := procDir.Readdir(-1)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, process := range processes {
+		if process.IsDir() {
+			pid, convErr := strconv.Atoi(process.Name())
```

Comment:
Could we utilize [GetProcessPID](https://github.com/longhorn/go-common-libs/blob/main/proc/proc.go#L58)?